docs/USAGE-GUIDE.md

```diff
@@ -703,8 +715,6 @@ or low latency. Connections prefering low latency will be encrypted using small
 record sizes that can be decrypted sooner by the recipient. Connections
 prefering throughput will use large record sizes that minimize overhead.
 
-Connections prefer low latency by default.
```

Comment:
If we're going to be specific(the previous docs were purposefully vague), `Connections default to an 8k outgoing maximum` is better imo.